src/vm/gdbjit.cpp

```diff
@@ -1354,6 +1386,9 @@ bool NotifyGdb::BuildDebugInfo(MemBuf& buf, NewArrayHolder<ArgsDebugInfo> &argsD
     DebugInfoVar* bufVar = new (nothrow) DebugInfoVar[localsDebugSize + argsDebugSize];
     if (bufVar == nullptr)
         return false;
+    char *bufVarLoc = new(nothrow)char[16];
```

Comment:
Is there a reason for allocating the buffer from the heap instead of having it on stack when it is that small?